registers/TUTORIAL.md

```diff
@@ -0,0 +1,16 @@
+Step 1
```

Comment:
Might be worth also having an overview of what you'll learn by following the tutorial?

Reply:
OK. Of course, at this point, it is somewhat aspirational!